<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-05T15:06:54 GMT-08:00" name="Default suite" finished-at="2023-11-05T15:06:56 GMT-08:00" duration-ms="1820">
    <groups>
    </groups>
    <test started-at="2023-11-05T15:06:54 GMT-08:00" name="Default test" finished-at="2023-11-05T15:06:56 GMT-08:00" duration-ms="1820">
      <class name="listeners.TestNgListenerDemo">
        <test-method signature="test1()[pri:0, instance:listeners.TestNgListenerDemo@3012646b]" started-at="2023-11-05T15:06:54 GMT-08:00" name="test1" finished-at="2023-11-05T15:06:54 GMT-08:00" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method signature="test2()[pri:0, instance:listeners.TestNgListenerDemo@3012646b]" started-at="2023-11-05T15:06:54 GMT-08:00" name="test2" finished-at="2023-11-05T15:06:56 GMT-08:00" duration-ms="1788" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method signature="test3()[pri:0, instance:listeners.TestNgListenerDemo@3012646b]" started-at="2023-11-05T15:06:56 GMT-08:00" name="test3" finished-at="2023-11-05T15:06:56 GMT-08:00" duration-ms="1" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[This test is skipped]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: This test is skipped
at listeners.TestNgListenerDemo.test3(TestNgListenerDemo.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
      </class> <!-- listeners.TestNgListenerDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
